### PR TITLE
Add acceptance tests to cover param for v-e-c Task

### PR DESCRIPTION
**Pull request by lcarva**
Ref: EC-334

### CODE REVIEW

File: acceptance/kubernetes/kind/kubernetes.go [hunk 1]
```diff
@@ -400,10 +400,17 @@ func (k *kindCluster) TaskInfo(ctx context.Context) (*types.TaskInfo, error) {
 			return nil, err
 		}
 
+		envVars, err := k.envVars(ctx, t.namespace, tr.Status.PodName, s.Container)
+		if err != nil {
+			return nil, err
+		}
+
+		// TODO: Grab env vars here (maybe from the pod?)
```
Comment (simonbaird) [inline acceptance/kubernetes/kind/kubernetes.go:408]: Is this todo stale now?
```suggestion
```

Comment (lcarva) [reply acceptance/kubernetes/kind/kubernetes.go]: Yes. Removed it. Thanks for catching this.

File: features/task_validate_image.feature [hunk 2]
```diff
@@ -259,3 +260,80 @@ Feature: Verify Enterprise Contract Tekton Tasks
...
+      And the task logs for step "report" should match the snapshot
+      And the task results should match the snapshot
+
+  Scenario: Effective-time is honored
+    Given a working namespace
+      And a key pair named "known"
+      And an image named "acceptance/effective-time"
+      And a valid image signature of "acceptance/effective-time" image signed by the "known" key
+      And a valid attestation of "acceptance/effective-time" signed by the "known" key
+      And a valid attestation of "acceptance/effective-time" signed by the "known" key
```
Comment (simonbaird) [inline features/task_validate_image.feature:289]: Are you creating two attestations intentionally, or is this line duplicated?

Comment (simonbaird) [reply features/task_validate_image.feature:289]: Also line 308 and 327. Maybe add a comment to explain why if it is intentional.

**Review by simonbaird — APPROVED** [review]

File: features/task_validate_image.feature (hunk 2 above)
Comment (lcarva) [reply features/task_validate_image.feature]: Nope, not intentional. My bad. Removed them.

**Pull request merged**